EnglishFilter/sections/specific.txt

```diff
@@ -169,6 +169,7 @@ watchonline.me##.remove-ad
 xda-developers.com##a[href="https://forum.xda-developers.com/getbrave"] > img
 rankedboost.com##.ArticleTopInsert
 rankedboost.com##div[class^="ArticleInsert"]
+healevate.com##.banner
```

Comment:
I think that this rule hides good content (articles in the right pane), so could you please check it again?
And maybe we can block the whole left pane `.paid-left`, marked red on screenshot.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/53902077-ad3c3700-4040-11e9-8ad9-f02c704531ea.png)

</details>

Reply:
I suggest these rules
```
healevate.com##.widget_text.banner
healevate.com##article[id^="post-"] div.banner
healevate.com#$#.paid-left { position: absolute!important; left: -3000px!important; }
```